config.yaml

```diff
@@ -8,3 +8,6 @@ options:
   metadata_url:
     type: string
     description: URL to the IdP's metadata
+  certificate:
+    type: string
+    description: Public certificate for the metadata URL validation
```

Comment:
So, if you don't want to validate, you just need to leave this config empty, right? If this is the case, it might be helpful to add a note about that here. What do you think?

Reply:
I think it's a good idea